torchtune/models/gemma/_model_builders.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import List, Optional
+from functools import partial
+
+from torch import nn
+
+from torchtune.models.gemma._component_builders import gemma, lora_gemma
+
+from torchtune.modules import Tokenizer, TransformerDecoder
+from torchtune.modules.peft import LORA_ATTN_MODULES
+
+"""
+Model builders build specific instantiations using component builders. For example
+the ``gemma_2b`` model builder uses the ``gemma`` component builder.
+"""
+
+
+def gemma_2b() -> TransformerDecoder:
+    """
+    Builder for creating a Gemma 2B model initialized w/ the default 2b parameter values
+    from: https://blog.google/technology/developers/gemma-open-models/
+
+    Returns:
+        TransformerDecoder: Instantiation of Gemma 2B model
+    """
+    return gemma(
+        vocab_size=256_000,
+        num_layers=18,
+        num_heads=8,
+        head_dim=256,
+        num_kv_heads=1,
+        embed_dim=2048,
+        intermediate_dim=16384,
+        max_seq_len=32768,
```

Comment:
Is this right? I thought this was 8192 for Gemma 2B

Reply:
Sorry, this is my problem, I fixed it in the latest version.